config/crd/bases/cloudsql.cloud.google.com_authproxyworkloads.yaml

```diff
@@ -30,7 +30,7 @@ spec:
     - name: v1alpha1
       schema:
         openAPIV3Schema:
-          description: AuthProxyWorkload declares how a Cloud SQL Proxy container should be applied to a matching set of workloads, and shows the status of those proxy containers. This is the Schema for the authproxyworkloads API.
+          description: AuthProxyWorkload declares how a Cloud SQL Proxy container should be applied to a matching set of workloads, and shows the status of those proxy containers.
```

Comment:
All changes in this file are generated by controller-gen from the godoc in `authproxyworkload_types.go`